pages/service/[id].tsx

```diff
@@ -208,6 +224,17 @@ export const ServicePage = ({ serviceData }: ServicePageProps): JSX.Element => {
         ) : null}
       </Section>
 
+      {serviceData.quote ? (
+        <Section divider={MyBestLifeTheme.colours.aqua}>
+          <Heading as="h2">What do other young people say?</Heading>
+          <Quote>
+            <QuoteMark side={'left'}>&ldquo;</QuoteMark>
+            <blockquote>{serviceData.quote}</blockquote>
+            <QuoteMark side={'right'}>&rdquo;</QuoteMark>
+          </Quote>
```

Comment:
Could be achieved with CSS `quotes` + pseudo content
https://developer.mozilla.org/en-US/docs/Web/CSS/quotes

This would allow you to reduce this UI to a single component/HTML element.

Reply:
I intended these to be images that I didn't have in high quality at the time, but they could still be pseudo content. I'll add this to a couple of other image fixes I need to do 👍